src/types/ExtraResponseAttributes.ts

```diff
@@ -0,0 +1,6 @@
+/**
+ * Represents a SASjs request parameter that will select which attributes are wanted in
+ * response object on top of default `webout` object.
+ *
+ */
+export type ExtraResponseAttributes = 'file' | 'data' | 'output' | null
```

Comment:
shouldn't 'result' be one of these outputs?  Unless we rename it 'webout'?

Reply:
hm, we did not clarify it, my idea was `result + these attributes here`